projects/admin/src/app/record/editor/type/repeat-section.type.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019-2023 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component } from '@angular/core';
+import { FieldArrayType } from '@ngx-formly/core';
+
+@Component({
+  selector: 'admin-repeat-section',
+  template: `
+    <div *ngFor="let field of field.fieldGroup; let i = index;" class="row">
+      <div class="col-11">
+        <formly-field [field]="field"></formly-field>
+      </div>
+      <div class="my-0 d-flex align-items-center">
```

Comment:
Cosmetic : maybe just add `col` class on this tag to understand this is a column of the row